core/commands/storage/store.go

```diff
@@ -2,7 +2,7 @@ package storage
 
 import (
```

Comment:
core/commands/storage/hosts.go → core/commands/storage/hosts_cmd.go

It should be called hosts.go? we are getting rid of the *_cmd.go syntax?